roles/percona-server/tasks/main.yml

```diff
@@ -66,6 +66,11 @@
 - name: install root .my.cnf defaults file
   template: src=root/.my.cnf dest=/root/.my.cnf owner=root group=root mode=0600
 
+- name: remove mysql test database
+  mysql_db: state=absent name=test
+  notify:
+    - restart mysql server
```

Comment:
good observation. I thought it was just in good practice but i will make those changes and re-commit